code/LeftAndMain.php

```diff
@@ -429,12 +430,16 @@ public function schema(HTTPRequest $request): HTTPResponse
     /**
      * Creates a successful json response
      */
-    protected function jsonSuccess(int $statusCode, array $data = []): HTTPResponse
+    protected function jsonSuccess(int $statusCode, ?array $data = null): HTTPResponse
     {
         if ($statusCode < 200 || $statusCode >= 300) {
             throw new InvalidArgumentException("Status code $statusCode must be between 200 and 299");
         }
-        $body = empty($data) ? '' : json_encode($data, JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE);
+        if (is_null($data)) {
+            $body = '';
+        } else {
+            $body = json_encode($data, JSON_UNESCAPED_SLASHES | JSON_UNESCAPED_UNICODE);
+        }
```

Comment:
This change doesn't seem to actually do anything? Doesn't it give the exact same result it did before?

Reply:
It's to allow JSON encoding an empty array. Previously it would return an empty body instead.

Reply:
See https://github.com/silverstripe/silverstripe-admin/pull/1802#discussion_r1726207432